distributed/deploy/local.py

```diff
@@ -133,6 +137,13 @@ def __init__(
             )
             threads_per_worker = None
 
+        if "dashboard" in worker_kwargs:
+            warnings.warn(
+                "Setting `dashboard` is discouraged. "
+                "Please set `worker_dashboard_address` or `dashboard_address` to "
+                "get best behavior."
```

Comment:
worker_dashboard_address here is a normal kwarg, while dashboard_address would refer to the scheduler's dashboard address I think if it was used as a normal kwarg, rather than within worker_kwargs.

Reply:
Right that's why `dashboard` is potentially ambiguous. Do they mean to pass `dashboard` to `work_kwargs` or to scheduler?

Reply:
Ah.  I understood this issue, but misunderstood your recommendation because I didn't realize that `**worker_kwargs` was splatted.  I felt like we were encouraging them to say `worker_dashboard_address=...` or `worker_kwargs={"dashboard_address": ...}`.  

I suspect that 99% of the time that this is used the user will intend to affect the scheduler dashboard.  Because of that we might start with `dashboard_address` rather than `worker_dashboard_address`, maybe calling out something like "use the dashboard_address= keyword to affect the scheduler and the worker_dashboard_address= keyword for the worker"

Reply:
Yeah that makes sense. I added more common/less common to try to nip that in the bud.